bundle/regal/rules/style/yoda_condition.rego

```diff
@@ -0,0 +1,30 @@
+# METADATA
+# description: Yoda condition
+package regal.rules.style["yoda-condition"]
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.regal.ast
+import data.regal.result
+
+report contains violation if {
+	walk(input.rules, [_, value])
+
+	value[0].type == "ref"
+	value[0].value[0].type == "var"
+	value[0].value[0].value == "equal"
```

Comment:
Should we do the same for `neq`?

Reply:
Hehe yeah I guess that's even more devilish :) I think all "operators" are reasonable to include here, but we can scope this to equality for now. I'll add `neq` too.